test/run_tests.sh

```diff
@@ -94,30 +95,42 @@ function run_xla_backend_mp {
   MASTER_ADDR=localhost MASTER_PORT=6000 run_test "$@"
 }
 
-function run_pjrt {
-  echo "Running in PjRt runtime: $@"
+function run_xrt {
   if [ -x "$(command -v nvidia-smi)" ]; then
-    PJRT_DEVICE=GPU run_test "$@"
+    GPU_NUM_DEVICES=2 "$@"
   else
-    # TODO(darisoy): run these tests with multiple CPU devices, this fails due to TF issue.
-    PJRT_DEVICE=CPU CPU_NUM_DEVICES=1 run_test "$@"
+    XRT_DEVICE_MAP="CPU:0;/job:localservice/replica:0/task:0/device:XLA_CPU:0" XRT_WORKERS="localservice:0;grpc://localhost:$(shuf -i 40701-40999 -n 1)" "$@"
   fi
 }
 
+function run_opbyop {
+  echo "Running in OpByOp mode: $@"
+  XLA_GET_TENSORS_OPBYOP=1 XLA_SYNC_TENSORS_OPBYOP=1 run_xrt "$@"
+}
+
 function run_async_scalar {
   echo "Running in Async Scalar Upload mode: $@"
-  XLA_TRANSFER_SCALAR_ASYNC=1 run_test "$@"
+  XLA_TRANSFER_SCALAR_ASYNC=1 run_xrt "$@"
 }
 
 function run_torchrun {
   echo "Running tests spawned by torchrun"
   if [ -x "$(command -v nvidia-smi)" ]; then
-    run_test "$@"
+    run_xrt "$@"
   else
     echo "the tests need atleast two XLA workers to validate"
   fi
 }
 
+function run_xrt_tests {
+  echo "Running XRT tests"
+  run_opbyop python3 "$CDIR/test_operations.py" "$@" --verbosity=$VERBOSITY
+  run_async_scalar python3 "$CDIR/test_operations.py" "$@" --verbosity=$VERBOSITY
+  run_xrt python3 "$CDIR/test_torch_distributed_xla_backend.py"
+  run_xrt python3 "$CDIR/test_mp_rendezvous.py"
```

Comment:
do we know why these two test failed on PJRT?

Reply:
Rendezvous uses replica groups, which we don't support with the new rendezvous implementation.

The distributed backend test relies on some implementation details of XRT that don't apply to PJRT. I can create a PJRT version, since `torch.distributed` largely works.